pypeit/flatfield.py

```diff
@@ -961,10 +980,13 @@ def fit(self, spat_illum_only=False, doqa=True, debug=False):
             # TODO: Put this stuff in a self.spectral_fit method?
 
             # Create the tilts image for this slit
-            # TODO -- JFH Confirm the sign of this shift is correct!
-            _flexure = 0. if self.wavetilts.spat_flexure is None else self.wavetilts.spat_flexure
-            tilts = tracewave.fit2tilts(rawflat.shape, self.wavetilts['coeffs'][:,:,slit_idx],
-                                        self.wavetilts['func2d'], spat_shift=-1*_flexure)
+            if self.slitless:
+                tilts = np.outer(np.arange(rawflat.shape[0]) / rawflat.shape[0], np.ones(rawflat.shape[1]))
```

Comment:
Don't use `np.outer`.  Use `np.tile` instead.

Reply:
I gotta learn that method someday